### PR TITLE
Add doc for inspect swarm

### PR DESCRIPTION
**Pull request by eskaaren**
**\- What I did**
Needed the swarm version number, no doc for doing swarm inspect existed. Tried get /swarm and it worked.

**\- How I did it**
GET /swarm

**\- How to verify it**
GET /swarm

**\- Description for the changelog**
Add documentation for swarm inspect via API.

Signed-off-by: Eivin Giske Skaaren eivin@sysmystic.com

### CODE REVIEW

File: docs/reference/api/docker_remote_api_v1.24.md [hunk 1]
```diff
@@ -4085,6 +4085,54 @@ JSON Parameters:
...
+
+    {
+      "CreatedAt" : "2016-08-15T16:00:20.349727406Z",
+      "Spec" : {
+        "Dispatcher" : {
+          "HeartbeatPeriod" : 5000000000
+        },
+        "Orchestration" : {
+         "TaskHistoryRetentionLimit" : 10
+  \     },
```
Comment (thaJeztah) [inline docs/reference/api/docker_remote_api_v1.24.md]: Looks like there's a stray backslash here

File: docs/reference/api/docker_remote_api_v1.25.md [hunk 2]
```diff
@@ -4109,6 +4109,54 @@ JSON Parameters:
...
+
+    {
+      "CreatedAt" : "2016-08-15T16:00:20.349727406Z",
+      "Spec" : {
+        "Dispatcher" : {
+          "HeartbeatPeriod" : 5000000000
+        },
+        "Orchestration" : {
+         "TaskHistoryRetentionLimit" : 10
+  \     },
```
Comment (thaJeztah) [inline docs/reference/api/docker_remote_api_v1.25.md]: And here

**Comment by vdemeester** [comment]
LGTM :cat:

**Comment by thaJeztah** [comment]
LGTM, thanks for fixing this!

**Pull request merged**